micronota/db/uniref.py

```diff
@@ -314,52 +220,55 @@ def prepare_metadata(in_fp, out_fp, append=True):
     The schema of the database file contains one table named `tigrfam` that
     has following columns:
 
-    1. ``ac``. TEXT. UniProtKB accession.
-
-    2. ``dataset``. TEXT.
+    1. ``ac``. TEXT. UniProtKB primary accession.
 
-    3. ``taxon``. TEXT.
+    2. ``status``. INT. The index in ``_status``
 
-    4. ``transfer``. INTEGER. Used as the boolean. ``1`` means the ``val``
-       should be transferred to the query sequences as its annotation;
-       ``0`` means not.
+    3. ``kingdom``. INT. The index in ``_kingdom``
 
     The table in the database file will be dropped and re-created if
     the function is re-run.
     '''
-    _overwrite(out_fp, append=append)
-    with connect(out_fp) as conn:
+    _overwrite(db_fp, **kwargs)
+    status_map = {k: i for i, k in enumerate(_status)}
+    kingdom_map = {k: i for i, k in enumerate(_kingdom)}
+    # this is the namespace for uniprot xml files.
+    ns_map = {'xmlns': 'http://uniprot.org/uniprot',
+              'xsi': 'http://www.w3.org/2001/XMLSchema-instance'}
```

Comment:
what are you referring to? `ns_map`? it is not about url and downloading. just the name space specified inside the xml file.